examples/stream_calc/container.py

```diff
@@ -14,17 +14,24 @@
 # limitations under the License.
 #
 
+# When mypy is started from pre-commit, it exits with an exception when analysing this
+# file without useful error message, thus ignoring.
+# (Please note this does not occur when running mypy directly.)
+#  type: ignore
```

Comment:
Thanks for investigating this.

Yes having a dedicated mypy.ini would be great. It was previously prevent by [this bug](https://github.com/python/mypy/issues/9533). But since the fix is released now, we can give it another try.